src/test/kotlin/org/opensearch/replication/integ/rest/StopReplicationIT.kt

```diff
@@ -70,6 +70,7 @@ class StopReplicationIT: MultiClusterRestTestCase() {
         }
     }
 
+    @AwaitsFix(bugUrl = "")
```

Comment:
Would this disable the tests? Is it because this test is flaky?

Reply:
We have a tracking issue on the repo. Can we rebase from main?